packages/formation-react/src/components/SegmentedProgressBar/SegmentedProgressBar.stories.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import SegmentedProgressBar from './SegmentedProgressBar';
+
+export default {
+  title: 'Library/SegmentedProgressBar',
+  component: SegmentedProgressBar,
+  decorators: [
+    Story => (
+      <div style={{ margin: '3em' }}>
+        <Story />
+      </div>
+    ),
+  ],
+
+  parameters: {
+    docs: {
+      description: {
+        component: 'Create a segmented progress bar for multi-page forms',
+      },
+    },
+  },
```

Comment:
I'm not a _huge_ fan of this and am ok removing it. This takes priority over the documentation generated from the JSDoc comments on the function, which would include some JSDoc-specific decorators which storybook doesn't interpret well:

https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/50d298a11261d270559fcca7c7a1544de29de4e5/packages/formation-react/src/components/SegmentedProgressBar/SegmentedProgressBar.jsx#L5-L10

![image](https://user-images.githubusercontent.com/2008881/99013640-c8459680-2505-11eb-9119-d915e757e2cc.png)


We may want those annotations in there for code editors to use :shrug: 

Right now there doesn't seem to be a low effort way to leave the comments in _and_ have storybook use the function comment but ignore the JSDoc annotations. I may write up an issue on the storybook add-on repo and see if there is any interest in allowing this kind of thing to happen.